From c27c1cdaf115197170a75f66cd2f9b92881427bc Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Tue, 22 Nov 2016 11:13:52 +0200
Subject: [PATCH 0605/1345] fix: dts: u3d: add missing SerDes-PHY#1 address
 register

commit  3320b0da7bfb289cf29f1c957deeb2991c9eabfb from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Add missing SerDes-PHY#1 address register for VBUS change,
if USB device is connected to another lane, the base register
should be updated due SerDes PHY number

Change-Id: I1131ed5a649183c859a82deb2546541e3e5c23a7
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/33952
Reviewed-by: Omri Itach <omrii@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-cp110.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
index 7555e99..d3a4a0f 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
@@ -151,7 +151,7 @@ u3d@520000 {
 	/* 0: 0x520000: USB 3.0 Device port 0: DEV_INFO_REG(0:15 - version_id)
 	 * 1: 0x121F10: Clock Source High reg: VBUS present indication in PHY SerDes lane#1
 	 */
-	reg = <0x520000 0x2000>;
+	reg = <0x520000 0x2000>, <0x121f10 0x40>;
 	interrupts = <GIC_SPI 60 IRQ_TYPE_LEVEL_HIGH>;
 	clocks = <&cpm_syscon0 1 24>;
 	dma-coherent;
-- 
1.7.9.5

